locales/ko_KR/LC_MESSAGES/duckduckgo.po

```diff
@@ -658,22 +658,22 @@ msgstr "가운데 정렬"
 #. http://i.imgur.com/CklQ1Ku.png
 msgctxt "settings"
 msgid "Center align the results page (instead of left aligned)"
-msgstr "검색 결과 페이지 가운데 정렬 (기본값은 왼쪽 정렬)"
+msgstr "검색 결과 페이지를 가운데 정렬합니다. (기본값은 왼쪽 정렬입니다)"
 
 msgid "Change Region"
-msgstr "지역 바꾸기"
+msgstr "지역 변경하기"
 
 #. http://i.imgur.com/OntTi6e.png
 msgctxt "settings"
 msgid "Change the background color across the entire site"
-msgstr "전체 사이트에 적용하는 배경색 바꾸기"
+msgstr "사이트 전체에 적용할 배경색을 변경합니다."
```

Comment:
I'm curious, is there a language-specific reason why you added periods at the end of sentences when not present in the original string? It is usually better to maintain the punctuation of the original string, unless the language specifically calls for something different. Let me know your thoughts.

Reply:
I thought it would sound better to write full sentences instead of the current phrases, but you have a point there. I will make sure to edit them with proper punctuations and push a new commit. Thanks!